src/Cards/CartCard.js

```diff
@@ -0,0 +1,82 @@
+import "./ProductCard.css";
+import { useMyReducer } from "../stateContext";
+import { BsTrashFill } from "react-icons/bs";
+import { BsPlus } from "react-icons/bs";
+import { FiMinus } from "react-icons/fi";
+import { Link } from "react-router-dom";
+import { useLangContext } from "../AllContext/languageContext";
+export const CartCard = (item) => {
+  const {
+    id,
+    imageSrc,
+    productName,
+    description,
+    price,
+    originalPrice,
+    newFlag,
+    quantity
+  } = item;
+  const { dispatch } = useMyReducer();
+  const { language } = useLangContext();
+  return (
+    <div className="card">
+      <div className="product-details">
+        <Link to={`/products/${id}`}>
+          <img className="product-image" alt="product" src={imageSrc} />
+          {newFlag && <div className="float-new">{language.new}</div>}
+        </Link>
+        <div className="product-name-and-desc">
+          <Link to={`/products/${id}`}>
+            <h3 style={{ margin: 0 }}>{productName}</h3>
```

Comment:
you can move the styling to a CSS class for h3 & re-use it if required.

Reply:
Done this